packages/cursorless-engine/src/core/HatTokenMapImpl.ts

```diff
@@ -106,8 +104,8 @@ export class HatTokenMapImpl implements HatTokenMap {
       }
 
       if (
-        abs(hrtime.bigint() - this.prePhraseMapsSnapshotTimestamp!) >
-        PRE_PHRASE_SNAPSHOT_MAX_AGE_NS
+        Math.abs(Date.now() - this.prePhraseMapsSnapshotTimestamp!) >
```

Comment:
Do we actually need abs here? The time now can never be before the timestamp we saved earlier?